INFO - train - Running command 'train'
INFO - train - Started run with ID "4"
{'exp_name': 'default', 'display': False, 'restore_fp': None, 'save_rate': 10, 'scenario_name': 'simple_spread', 'num_episodes': 60000, 'max_episode_len': 25, 'good_policy': 'matd3', 'adv_policy': 'matd3', 'lr': 0.01, 'gamma': 0.95, 'batch_size': 1024, 'num_layers': 2, 'num_units': 64, 'update_rate': 100, 'critic_zero_if_done': False, 'buff_size': 1000000.0, 'tau': 0.01, 'hard_max': False, 'priori_replay': False, 'alpha': 0.6, 'beta': 0.5, 'use_target_action': True, 'policy_update_rate': 1, 'critic_action_noise_stddev': 0.0, 'action_noise_clip': 0.5, 'entropy_coeff': 0.05, 'num_atoms': 51, 'min_val': -400, 'max_val': 0, 'seed': 124391599}
2021-04-07 01:31:12.009583: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN)to use the following CPU instructions in performance-critical operations:  AVX2 FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2021-04-07 01:31:12.036414: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x7ff921f773b0 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2021-04-07 01:31:12.036432: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
Using good policy matd3 and adv policy matd3
4
Starting iterations...
WARNING:tensorflow:5 out of the last 5 calls to <function MADDPGPolicyNetwork.get_action at 0x1ef2063488> triggered tf.function retracing. Tracing is expensive and the excessive number of tracings could be due to (1) creating @tf.function repeatedly in a loop, (2) passing tensors with different shapes, (3) passing Python objects instead of tensors. For (1), please define your @tf.function outside of the loop. For (2), @tf.function has experimental_relax_shapes=True option that relaxes argument shapes that can avoid unnecessary retracing. For (3), please refer to https://www.tensorflow.org/tutorials/customization/performance#python_or_tensor_args and https://www.tensorflow.org/api_docs/python/tf/function for  more details.
WARNING - tensorflow - 5 out of the last 5 calls to <function MADDPGPolicyNetwork.get_action at 0x1ef2063488> triggered tf.function retracing. Tracing is expensive and the excessive number of tracings could be due to (1) creating @tf.function repeatedly in a loop, (2) passing tensors with different shapes, (3) passing Python objects instead of tensors. For (1), please define your @tf.function outside of the loop. For (2), @tf.function has experimental_relax_shapes=True option that relaxes argument shapes that can avoid unnecessary retracing. For (3), please refer to https://www.tensorflow.org/tutorials/customization/performance#python_or_tensor_args and https://www.tensorflow.org/api_docs/python/tf/function for  more details.
WARNING:tensorflow:6 out of the last 6 calls to <function MADDPGPolicyNetwork.get_action at 0x1ef2063f28> triggered tf.function retracing. Tracing is expensive and the excessive number of tracings could be due to (1) creating @tf.function repeatedly in a loop, (2) passing tensors with different shapes, (3) passing Python objects instead of tensors. For (1), please define your @tf.function outside of the loop. For (2), @tf.function has experimental_relax_shapes=True option that relaxes argument shapes that can avoid unnecessary retracing. For (3), please refer to https://www.tensorflow.org/tutorials/customization/performance#python_or_tensor_args and https://www.tensorflow.org/api_docs/python/tf/function for  more details.
WARNING - tensorflow - 6 out of the last 6 calls to <function MADDPGPolicyNetwork.get_action at 0x1ef2063f28> triggered tf.function retracing. Tracing is expensive and the excessive number of tracings could be due to (1) creating @tf.function repeatedly in a loop, (2) passing tensors with different shapes, (3) passing Python objects instead of tensors. For (1), please define your @tf.function outside of the loop. For (2), @tf.function has experimental_relax_shapes=True option that relaxes argument shapes that can avoid unnecessary retracing. For (3), please refer to https://www.tensorflow.org/tutorials/customization/performance#python_or_tensor_args and https://www.tensorflow.org/api_docs/python/tf/function for  more details.
WARNING:tensorflow:7 out of the last 7 calls to <function MADDPGPolicyNetwork.get_action at 0x1ef2074ae8> triggered tf.function retracing. Tracing is expensive and the excessive number of tracings could be due to (1) creating @tf.function repeatedly in a loop, (2) passing tensors with different shapes, (3) passing Python objects instead of tensors. For (1), please define your @tf.function outside of the loop. For (2), @tf.function has experimental_relax_shapes=True option that relaxes argument shapes that can avoid unnecessary retracing. For (3), please refer to https://www.tensorflow.org/tutorials/customization/performance#python_or_tensor_args and https://www.tensorflow.org/api_docs/python/tf/function for  more details.
WARNING - tensorflow - 7 out of the last 7 calls to <function MADDPGPolicyNetwork.get_action at 0x1ef2074ae8> triggered tf.function retracing. Tracing is expensive and the excessive number of tracings could be due to (1) creating @tf.function repeatedly in a loop, (2) passing tensors with different shapes, (3) passing Python objects instead of tensors. For (1), please define your @tf.function outside of the loop. For (2), @tf.function has experimental_relax_shapes=True option that relaxes argument shapes that can avoid unnecessary retracing. For (3), please refer to https://www.tensorflow.org/tutorials/customization/performance#python_or_tensor_args and https://www.tensorflow.org/api_docs/python/tf/function for  more details.
WARNING:tensorflow:8 out of the last 8 calls to <function MADDPGPolicyNetwork.get_action at 0x1ef208a6a8> triggered tf.function retracing. Tracing is expensive and the excessive number of tracings could be due to (1) creating @tf.function repeatedly in a loop, (2) passing tensors with different shapes, (3) passing Python objects instead of tensors. For (1), please define your @tf.function outside of the loop. For (2), @tf.function has experimental_relax_shapes=True option that relaxes argument shapes that can avoid unnecessary retracing. For (3), please refer to https://www.tensorflow.org/tutorials/customization/performance#python_or_tensor_args and https://www.tensorflow.org/api_docs/python/tf/function for  more details.
WARNING - tensorflow - 8 out of the last 8 calls to <function MADDPGPolicyNetwork.get_action at 0x1ef208a6a8> triggered tf.function retracing. Tracing is expensive and the excessive number of tracings could be due to (1) creating @tf.function repeatedly in a loop, (2) passing tensors with different shapes, (3) passing Python objects instead of tensors. For (1), please define your @tf.function outside of the loop. For (2), @tf.function has experimental_relax_shapes=True option that relaxes argument shapes that can avoid unnecessary retracing. For (3), please refer to https://www.tensorflow.org/tutorials/customization/performance#python_or_tensor_args and https://www.tensorflow.org/api_docs/python/tf/function for  more details.
WARNING:tensorflow:9 out of the last 9 calls to <function MADDPGPolicyNetwork.get_action at 0x1ef20a0268> triggered tf.function retracing. Tracing is expensive and the excessive number of tracings could be due to (1) creating @tf.function repeatedly in a loop, (2) passing tensors with different shapes, (3) passing Python objects instead of tensors. For (1), please define your @tf.function outside of the loop. For (2), @tf.function has experimental_relax_shapes=True option that relaxes argument shapes that can avoid unnecessary retracing. For (3), please refer to https://www.tensorflow.org/tutorials/customization/performance#python_or_tensor_args and https://www.tensorflow.org/api_docs/python/tf/function for  more details.
WARNING - tensorflow - 9 out of the last 9 calls to <function MADDPGPolicyNetwork.get_action at 0x1ef20a0268> triggered tf.function retracing. Tracing is expensive and the excessive number of tracings could be due to (1) creating @tf.function repeatedly in a loop, (2) passing tensors with different shapes, (3) passing Python objects instead of tensors. For (1), please define your @tf.function outside of the loop. For (2), @tf.function has experimental_relax_shapes=True option that relaxes argument shapes that can avoid unnecessary retracing. For (3), please refer to https://www.tensorflow.org/tutorials/customization/performance#python_or_tensor_args and https://www.tensorflow.org/api_docs/python/tf/function for  more details.
WARNING:tensorflow:10 out of the last 10 calls to <function MADDPGPolicyNetwork.get_action at 0x1ef20a0d90> triggered tf.function retracing. Tracing is expensive and the excessive number of tracings could be due to (1) creating @tf.function repeatedly in a loop, (2) passing tensors with different shapes, (3) passing Python objects instead of tensors. For (1), please define your @tf.function outside of the loop. For (2), @tf.function has experimental_relax_shapes=True option that relaxes argument shapes that can avoid unnecessary retracing. For (3), please refer to https://www.tensorflow.org/tutorials/customization/performance#python_or_tensor_args and https://www.tensorflow.org/api_docs/python/tf/function for  more details.
WARNING - tensorflow - 10 out of the last 10 calls to <function MADDPGPolicyNetwork.get_action at 0x1ef20a0d90> triggered tf.function retracing. Tracing is expensive and the excessive number of tracings could be due to (1) creating @tf.function repeatedly in a loop, (2) passing tensors with different shapes, (3) passing Python objects instead of tensors. For (1), please define your @tf.function outside of the loop. For (2), @tf.function has experimental_relax_shapes=True option that relaxes argument shapes that can avoid unnecessary retracing. For (3), please refer to https://www.tensorflow.org/tutorials/customization/performance#python_or_tensor_args and https://www.tensorflow.org/api_docs/python/tf/function for  more details.
Cost: -7.280668834146711
Cost: -7.261285988923844
Cost: -7.240903854250939
Cost: -7.268383159454112
Cost: -7.265478799493187
Cost: -7.225019287194285
Cost: -7.302904283159146
Cost: -7.273778347063498
Cost: -7.266048926840016
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.2411416206464345
Cost: -7.268340747949257
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241091732447301
Cost: -7.268341808859364
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241101471437129
Cost: -7.268341782526175
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.2410995415127966
Cost: -7.268341782769139
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099921210879
Cost: -7.268341782304703
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099847196135
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099861826264
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859025384
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
steps: 9, episodes: 10, mean episode reward: -72.62553401371395, agent episode reward: [-7.280668834261274, -7.261285988923843, -7.241103746082762, -7.268341670585282, -7.265478799493187, -7.225019021919476, -7.302904283159147, -7.271914266103581, -7.266048937263258, -7.242768465922138], time: 0.512
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
steps: 19, episodes: 20, mean episode reward: -72.62553023920316, agent episode reward: [-7.280668834261274, -7.261285988923843, -7.241099859442536, -7.268341782714724, -7.265478799493187, -7.225019021919476, -7.302904283159147, -7.271914266103581, -7.266048937263258, -7.242768465922138], time: 0.848
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
steps: 29, episodes: 30, mean episode reward: -72.62553023920316, agent episode reward: [-7.280668834261274, -7.261285988923843, -7.241099859442536, -7.268341782714724, -7.265478799493187, -7.225019021919476, -7.302904283159147, -7.271914266103581, -7.266048937263258, -7.242768465922138], time: 0.87
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
steps: 39, episodes: 40, mean episode reward: -72.62553023920316, agent episode reward: [-7.280668834261274, -7.261285988923843, -7.241099859442536, -7.268341782714724, -7.265478799493187, -7.225019021919476, -7.302904283159147, -7.271914266103581, -7.266048937263258, -7.242768465922138], time: 0.889
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
steps: 49, episodes: 50, mean episode reward: -72.62553023920316, agent episode reward: [-7.280668834261274, -7.261285988923843, -7.241099859442536, -7.268341782714724, -7.265478799493187, -7.225019021919476, -7.302904283159147, -7.271914266103581, -7.266048937263258, -7.242768465922138], time: 0.789
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
steps: 59, episodes: 60, mean episode reward: -72.62553023920316, agent episode reward: [-7.280668834261274, -7.261285988923843, -7.241099859442536, -7.268341782714724, -7.265478799493187, -7.225019021919476, -7.302904283159147, -7.271914266103581, -7.266048937263258, -7.242768465922138], time: 0.816
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
steps: 69, episodes: 70, mean episode reward: -72.62553023920316, agent episode reward: [-7.280668834261274, -7.261285988923843, -7.241099859442536, -7.268341782714724, -7.265478799493187, -7.225019021919476, -7.302904283159147, -7.271914266103581, -7.266048937263258, -7.242768465922138], time: 0.791
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
steps: 79, episodes: 80, mean episode reward: -72.62553023920316, agent episode reward: [-7.280668834261274, -7.261285988923843, -7.241099859442536, -7.268341782714724, -7.265478799493187, -7.225019021919476, -7.302904283159147, -7.271914266103581, -7.266048937263258, -7.242768465922138], time: 0.806
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
steps: 89, episodes: 90, mean episode reward: -72.62553023920316, agent episode reward: [-7.280668834261274, -7.261285988923843, -7.241099859442536, -7.268341782714724, -7.265478799493187, -7.225019021919476, -7.302904283159147, -7.271914266103581, -7.266048937263258, -7.242768465922138], time: 0.791
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
steps: 99, episodes: 100, mean episode reward: -72.62553023920316, agent episode reward: [-7.280668834261274, -7.261285988923843, -7.241099859442536, -7.268341782714724, -7.265478799493187, -7.225019021919476, -7.302904283159147, -7.271914266103581, -7.266048937263258, -7.242768465922138], time: 0.815
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
steps: 109, episodes: 110, mean episode reward: -72.62553023920316, agent episode reward: [-7.280668834261274, -7.261285988923843, -7.241099859442536, -7.268341782714724, -7.265478799493187, -7.225019021919476, -7.302904283159147, -7.271914266103581, -7.266048937263258, -7.242768465922138], time: 0.821
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
steps: 119, episodes: 120, mean episode reward: -72.62553023920316, agent episode reward: [-7.280668834261274, -7.261285988923843, -7.241099859442536, -7.268341782714724, -7.265478799493187, -7.225019021919476, -7.302904283159147, -7.271914266103581, -7.266048937263258, -7.242768465922138], time: 0.791
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
steps: 129, episodes: 130, mean episode reward: -72.62553023920316, agent episode reward: [-7.280668834261274, -7.261285988923843, -7.241099859442536, -7.268341782714724, -7.265478799493187, -7.225019021919476, -7.302904283159147, -7.271914266103581, -7.266048937263258, -7.242768465922138], time: 0.809
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
steps: 139, episodes: 140, mean episode reward: -72.62553023920316, agent episode reward: [-7.280668834261274, -7.261285988923843, -7.241099859442536, -7.268341782714724, -7.265478799493187, -7.225019021919476, -7.302904283159147, -7.271914266103581, -7.266048937263258, -7.242768465922138], time: 0.797
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
steps: 149, episodes: 150, mean episode reward: -72.62553023920316, agent episode reward: [-7.280668834261274, -7.261285988923843, -7.241099859442536, -7.268341782714724, -7.265478799493187, -7.225019021919476, -7.302904283159147, -7.271914266103581, -7.266048937263258, -7.242768465922138], time: 0.775
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
Cost: -7.280668834261274
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.261285988923844
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
steps: 159, episodes: 160, mean episode reward: -72.62553023920316, agent episode reward: [-7.280668834261274, -7.261285988923843, -7.241099859442536, -7.268341782714724, -7.265478799493187, -7.225019021919476, -7.302904283159147, -7.271914266103581, -7.266048937263258, -7.242768465922138], time: 0.973
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
steps: 169, episodes: 170, mean episode reward: -72.62553023920316, agent episode reward: [-7.280668834261274, -7.261285988923843, -7.241099859442536, -7.268341782714724, -7.265478799493187, -7.225019021919476, -7.302904283159147, -7.271914266103581, -7.266048937263258, -7.242768465922138], time: 1.039
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
Cost: -7.280668834261274
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
steps: 179, episodes: 180, mean episode reward: -72.62553023920316, agent episode reward: [-7.280668834261274, -7.261285988923843, -7.241099859442536, -7.268341782714724, -7.265478799493187, -7.225019021919476, -7.302904283159147, -7.271914266103581, -7.266048937263258, -7.242768465922138], time: 1.055
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
Cost: -7.266048937263258
Cost: -7.242768465922138
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
Cost: -7.280668834261274
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
steps: 189, episodes: 190, mean episode reward: -72.62553023920316, agent episode reward: [-7.280668834261274, -7.261285988923843, -7.241099859442536, -7.268341782714724, -7.265478799493187, -7.225019021919476, -7.302904283159147, -7.271914266103581, -7.266048937263258, -7.242768465922138], time: 0.975
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
Cost: -7.242768465922138
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
steps: 199, episodes: 200, mean episode reward: -72.62553023920316, agent episode reward: [-7.280668834261274, -7.261285988923843, -7.241099859442536, -7.268341782714724, -7.265478799493187, -7.225019021919476, -7.302904283159147, -7.271914266103581, -7.266048937263258, -7.242768465922138], time: 0.996
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.266048937263258
Cost: -7.242768465922138
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
steps: 209, episodes: 210, mean episode reward: -72.62553023920316, agent episode reward: [-7.280668834261274, -7.261285988923843, -7.241099859442536, -7.268341782714724, -7.265478799493187, -7.225019021919476, -7.302904283159147, -7.271914266103581, -7.266048937263258, -7.242768465922138], time: 0.965
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.265478799493187
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
steps: 219, episodes: 220, mean episode reward: -72.62553023920316, agent episode reward: [-7.280668834261274, -7.261285988923843, -7.241099859442536, -7.268341782714724, -7.265478799493187, -7.225019021919476, -7.302904283159147, -7.271914266103581, -7.266048937263258, -7.242768465922138], time: 0.992
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
Cost: -7.242768465922138
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
steps: 229, episodes: 230, mean episode reward: -72.62553023920316, agent episode reward: [-7.280668834261274, -7.261285988923843, -7.241099859442536, -7.268341782714724, -7.265478799493187, -7.225019021919476, -7.302904283159147, -7.271914266103581, -7.266048937263258, -7.242768465922138], time: 0.932
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
Cost: -7.242768465922138
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
steps: 239, episodes: 240, mean episode reward: -72.62553023920316, agent episode reward: [-7.280668834261274, -7.261285988923843, -7.241099859442536, -7.268341782714724, -7.265478799493187, -7.225019021919476, -7.302904283159147, -7.271914266103581, -7.266048937263258, -7.242768465922138], time: 0.977
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
steps: 249, episodes: 250, mean episode reward: -72.62553023920316, agent episode reward: [-7.280668834261274, -7.261285988923843, -7.241099859442536, -7.268341782714724, -7.265478799493187, -7.225019021919476, -7.302904283159147, -7.271914266103581, -7.266048937263258, -7.242768465922138], time: 0.999
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
Cost: -7.280668834261274
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
steps: 259, episodes: 260, mean episode reward: -72.62553023920316, agent episode reward: [-7.280668834261274, -7.261285988923843, -7.241099859442536, -7.268341782714724, -7.265478799493187, -7.225019021919476, -7.302904283159147, -7.271914266103581, -7.266048937263258, -7.242768465922138], time: 1.034
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
Cost: -7.280668834261274
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
Cost: -7.280668834261274
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
Cost: -7.266048937263258
Cost: -7.242768465922138
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
steps: 269, episodes: 270, mean episode reward: -72.62553023920316, agent episode reward: [-7.280668834261274, -7.261285988923843, -7.241099859442536, -7.268341782714724, -7.265478799493187, -7.225019021919476, -7.302904283159147, -7.271914266103581, -7.266048937263258, -7.242768465922138], time: 0.892
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.242768465922138
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
steps: 279, episodes: 280, mean episode reward: -72.62553023920316, agent episode reward: [-7.280668834261274, -7.261285988923843, -7.241099859442536, -7.268341782714724, -7.265478799493187, -7.225019021919476, -7.302904283159147, -7.271914266103581, -7.266048937263258, -7.242768465922138], time: 1.034
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.242768465922138
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.302904283159146
Cost: -7.271914266103581
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
Cost: -7.266048937263258
Cost: -7.242768465922138
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
Cost: -7.242768465922138
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
steps: 289, episodes: 290, mean episode reward: -72.62553023920316, agent episode reward: [-7.280668834261274, -7.261285988923843, -7.241099859442536, -7.268341782714724, -7.265478799493187, -7.225019021919476, -7.302904283159147, -7.271914266103581, -7.266048937263258, -7.242768465922138], time: 0.948
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
Cost: -7.280668834261274
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
Cost: -7.280668834261274
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
steps: 299, episodes: 300, mean episode reward: -72.62553023920316, agent episode reward: [-7.280668834261274, -7.261285988923843, -7.241099859442536, -7.268341782714724, -7.265478799493187, -7.225019021919476, -7.302904283159147, -7.271914266103581, -7.266048937263258, -7.242768465922138], time: 1.03
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
Cost: -7.265478799493187
Cost: -7.225019021919476
Cost: -7.302904283159146
Cost: -7.271914266103581
Cost: -7.266048937263258
Cost: -7.242768465922138
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/Users/dustin/anaconda3/envs/RAISE/lib/python3.6/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Cost: -7.280668834261274
Cost: -7.261285988923844
Cost: -7.241099859442536
Cost: -7.268341782714723
